docs/components/embedders.mdx

```diff
@@ -0,0 +1,80 @@
+---
+title: 🧩 Embedding models
+---
+
+## Overview
+
+Mem0 offers support for various embedding models, allowing users to choose the one that best suits their needs.
+
+<CardGroup cols={3}>
+    <Card title="OpenAI" href="#openai"></Card>
+    <Card title="Ollama" href="#ollama"></Card>
+    <Card title="LM Studio" href="#lm-studio"></Card>
+</CodeGroup>
+
+## OpenAI
+
+To use OpenAI embedding models, set the `OPENAI_API_KEY` environment variable. You can obtain the OpenAI API key from the [OpenAI Platform](https://platform.openai.com/account/api-keys).
+
+Example of how to select the desired embedding model:
+
+```python
+import os
+from mem0 import Memory
+
+config = {
+    "embedder": {
+        "provider": "openai",
+        "config": {
+            "model": "text-embedding-3-large"
+        }
+    }
+}
+
+m = Memory.from_config(config)
+m.add("I'm visiting Paris", user_id="john")
+```
+
+## Ollama
+
+You can use embedding models from Ollama to run Mem0 locally.
+
+Here's how to select it:
+
+``python
+import os
+from mem0 import Memory
+
+config = {
+    "embedder": {
+        "provider": "ollama",
+        "config": {
+            "model": "mxbai-embed-large"
+        }
+    }
+}
+
+m = Memory.from_config(config)
+m.add("I'm visiting Paris", user_id="john")
+```
+
+## LM Studio
+
+You can also utilize embedding models from LM Studio and where models can be directly downloaded from HuggingFace.
+
+``python
+import os
+from mem0 import Memory
+
+config = {
+    "embedder": {
+        "provider": "lmstudio",
+        "config": {
+            "model": "nomic-ai/nomic-embed-text-v1.5-GGUF/nomic-embed-text-v1.5.f16.gguf"
+        }
+    }
+}
+
+m = Memory.from_config(config)
+m.add("I'm visiting Paris", user_id="john")
+```
```

Comment:
Have you tried running these configs for embedders?
Especially for Ollama and LM Studio. I think they are error-prone.

Reply:
Hey I tested it out and it seems to have been working properly.

Reply:
Cool! I was just worried about the case with Qdrant as default db. Anyways it's updated in docs now! 
Thanks!